src/org/traccar/model/ExtendedModel.java

```diff
@@ -122,4 +122,10 @@ public long getLong(String key) {
         }
     }
 
+    public void remove(String key) {
```

Comment:
I first thought that after calling getLastPosition() that the attributes all came across, but could not find a way to remove a specific attribute. I dont use it now but cant see it being detrimental having this option.